firmware/quadruna/VC/src/hw/hw_adc.h

```diff
@@ -38,7 +38,8 @@ typedef enum
     ADC1_IN19_VBAT_SENSE,
     ADC3_IN0_AUX_PWR_I_SNS,
     ADC3_IN1_PUMP_PWR_I_SNS,
-    NUM_ADC_CHANNELS
+    NUM_ADC_CHANNELS,
+    NO_ADC // for cases where configuration requires an ADC pin, but there is none (TELEM/BUZZER efuses)
```

Comment:
nitpick: since C has no namespaces I think it's good practice to keep the name of the enum in each entry. Like `NO_ADC_CHANNEL` or `ADC_CHANNEL_UNAVAILABLE` for example.

Reply:
Fixed!